helm-charts/charts/kube-starrocks/charts/starrocks/values.yaml

```diff
@@ -61,7 +106,7 @@ starrocksFESpec:
   image:
     # image sliced by "repository:tag"
     repository: starrocks/fe-ubuntu
-    tag: 3.1-latest
+    tag: ""
```

Comment:
if we set it to empty string. will it also take precedence over the values in `componentValues`?

Reply:
No. Only when it has a non-empty value.